core/src/main/java/hudson/WebAppMain.java

```diff
@@ -80,8 +80,7 @@
  */
 public class WebAppMain implements ServletContextListener {
 
-    // use RingBufferLogHandler class name to configure for backward compatibility
-    private static final int DEFAULT_RING_BUFFER_SIZE = SystemProperties.getInteger(RingBufferLogHandler.class.getName() + ".defaultSize", 256);
+    private static final int DEFAULT_RING_BUFFER_SIZE = 256;
```

Comment:
Remove this. You can use the property in the `RingBufferLogHandler` which hold the value for every piece of code that needs it